bika/lims/content/abstractanalysis.py

```diff
@@ -195,6 +195,8 @@ def wasVerifiedByUser(self, username):
 
     @security.public
     def getLastVerificator(self):
+        if not self.getVerificators():
```

Comment:
Better this way:
```python
verificators = self.getVerificators()
if not verificators:
    return None
return verificators.split(',')[-1]
```

Also note the same problem happens in the following lines:
https://github.com/senaite/bika.lims/pull/419/files#diff-350aa20602ad7e16c9bfc735a3454256R186
https://github.com/senaite/bika.lims/pull/419/files#diff-350aa20602ad7e16c9bfc735a3454256R193

Apply the same fix for these two lines above, please.